website/docs/reference/commands/test.md

```diff
@@ -28,4 +29,48 @@ dbt test --select "one_specific_model,test_type:singular"
 dbt test --select "one_specific_model,test_type:generic"
 ```
 
-For more information on writing tests, see the [Testing Documentation](/docs/build/data-tests).
+For more information on writing tests, see the [Testing Documentation](/docs/build/tests).
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.8">
+
+`dbt test` runs data tests defined on models, sources, snapshots, and seeds and unit tests defined on SQL models. It expects that you have already created those resources through the appropriate commands.
```

Comment:
We need to update the `dbt build` command page as well

Reply:
(Can be a fast follow, but I don't want it to get forgotten!)

Reply:
Added it to the fast follow!